terminate called after throwing an instance of 'io::error::can_not_open_file'
  what():  Can not open file "test/test_csv_reader.csv" because "No such file or directory".
Aborted (core dumped)
